utils/navigator_tools/nodes/tf_to_gazebo.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+import roslib
+import rospy
+import math
+import tf
+import geometry_msgs.msg
+import numpy as np
+def main():
+    rospy.init_node('tf_to_gazebo')
+    do_cam_fix = rospy.get_param("~cam_fix",False)
+    listener = tf.TransformListener()
+    rate = rospy.Rate(10.0)
+    cam_fix_quat = (-0.5 , 0.5 ,-0.5 ,-0.5)
+
+    while not rospy.is_shutdown():
+        try:
+            print "============= TRANFORMS ================"
+            for frame_id in listener.getFrameStrings():
+                (trans,rot) = listener.lookupTransform('/measurement',frame_id, rospy.Time(0))
+                print "--"
+                print "Tranform {} {}".format('/measurement',frame_id)
```

Comment:
Transform is spelled wrong, also why not just put the string into the text `"Transform /measurement {}".format( ... )`